sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -425,10 +435,18 @@ public partial class ServiceBusSessionProcessor : System.IAsyncDisposable
         public override string ToString() { throw null; }
         public void UpdateConcurrency(int maxConcurrentSessions, int maxConcurrentCallsPerSession) { }
     }
+    public partial class ServiceBusSessionProcessorDiagnostics
+    {
+        protected ServiceBusSessionProcessorDiagnostics() { }
+        public System.DateTimeOffset? LastReceiveAttemptedTime { get { throw null; } }
+        public System.DateTimeOffset? LastReceiveSucceededTime { get { throw null; } }
+        public int MessagesBeingProcessed { get { throw null; } }
```

Comment:
Do we want this information per session?

Reply:
SessionsBeingProcessed